actionpack/lib/action_dispatch/http/resource_isolation_policy.rb

```diff
@@ -100,6 +104,10 @@ def call(env)
 
       private
 
+      def logger
+        ActiveSupport::Logger.new($stderr)
+      end
```

Comment:
Can you explore if this is covering all the cases or we should do sth like https://github.com/Zerocopter/rails/blob/master/actionpack/lib/action_dispatch/middleware/debug_exceptions.rb#L164-L170 ?

Reply:
Ah yes, that's exactly what I was looking for, thanks!